pkg/util/ipvs/kernel_check.go

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Please add an OWNERS file to this directory with the people you want to see there that own the ipvs stuff

Reply:
The OWNERS file is already there and I am one of the owners :)

I am fine with this change.

Reply:
@islinwb Can we reference this util package in proxy/ipvs?